backend/employees/views.py

```
@@ -1,3 +1,28 @@
+    serializer_class = TimeEntrySerializer
+    permission_classes = []
+
+    def get_queryset(self):
+        employee_id = self.kwargs.get("employee_pk")
+
+        if employee_id is not None:
+            try:
+                employee = Employees.objects.get(id=employee_id)
+                return TimeEntries.objects.filter(employee=employee)
+            except Employees.DoesNotExist:
+                raise NotFound(detail="Employee not found.")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)